src/MarkdownTextInput.web.tsx

```diff
@@ -546,6 +546,14 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
       updateSelection(null, {start: selection.start, end: selection.end || selection.start});
     }, [selection, updateSelection]);
 
+    useEffect(() => {
+      if (history.current?.history.length !== 0) {
+        return;
+      }
+      history.current.add(value ?? '', (value ?? '').length);
```

Comment:
```suggestion
      if (!value || history.current?.history.length !== 0) {
        return;
      }
      history.current.add(value, value.length);
```

Reply:
This wouldn't work, I actually do want the history item to be added if value is `undefined`, or just an empty string - that's what we're fixing here

Reply:
Right, so maybe let's move `value ?? ''` to variable since we are using it two times